src/teamforge.iced

```diff
@@ -801,18 +811,28 @@ class Team
   #-------------------
 
   _next_seqno : ->
+    @_last_seqno() + 1
+
+  #-------------------
+
+  _last_seqno : ->
     if @links.length > 0
-      @links[@links.length-1].for_client.seqno + 1
+      @links[@links.length-1].for_client.seqno 
     else
-      1
+      0
```

Comment:
doesn't this change the previous compilation output?

Reply:
I don't think so, maybe I am missing something? 
Spot checking the output here https://github.com/keybase/keybase-test-vectors/pull/27/ also seems to be ok.